3rdparty/haskell/unix-compat.patch

```diff
@@ -0,0 +1,14 @@
+--- src/System/PosixCompat/Files.hsc    2018-08-23 02:44:16.000000000 +0200
```

Comment:
This seems an odd place to put the patch. Should we have a `patches` directory somewhere? I'd much rather all our patches were in one place.

Reply:
Good point, I've moved it to where all the other Bazel related patches are.